enderio-base/src/main/java/com/enderio/base/common/block/glass/GlassCollisionPredicate.java

```diff
@@ -18,17 +18,17 @@
 public enum GlassCollisionPredicate {
     NONE(ctx -> false, null),
 
-    PLAYERS_PASS(ctx -> ctx.getEntity().map(ent -> ent instanceof Player).orElse(false), EIOLang.GLASS_COLLISION_PLAYERS_PASS),
+    PLAYERS_PASS(ctx -> ctx.getEntity() instanceof Player, EIOLang.GLASS_COLLISION_PLAYERS_PASS),
 
-    PLAYERS_BLOCK(ctx -> ctx.getEntity().map(ent -> !(ent instanceof Player)).orElse(true), EIOLang.GLASS_COLLISION_PLAYERS_BLOCK),
+    PLAYERS_BLOCK(ctx -> !(ctx.getEntity() instanceof Player), EIOLang.GLASS_COLLISION_PLAYERS_BLOCK),
```

Comment:
only as something to think about:
```suggestion
    PLAYERS_BLOCK(ctx -> !PLAYERS_PASS.canPass(ctx), EIOLang.GLASS_COLLISION_PLAYERS_BLOCK),
```
what if the block enumVars return the opposite of the pass enumVars. I think that's a bit more readable

Reply:
I like this, good call :)

Reply:
aw no, we're not allowed to do that. We're self-referencing in a constructor >:(